test/framework/integration_framework/integration_test_framework.cpp

```diff
@@ -312,6 +312,7 @@ IntegrationTestFramework::IntegrationTestFramework(
   config_.stale_stream_max_rounds = 2;
   config_.max_proposal_size = 10;
   config_.mst_support = mst_support;
+  config_.max_past_created_time = 24;
```

Comment:
```suggestion
  config_.max_past_created_hours = 24;
```

Reply:
I renamed everywhere (according to CTRL + F)